### DISCUSSION TITLE
Okay this is gonna be a revolutionary, gaming-changing, mind-blowing, MEGA-ultimate feature request that's never before been requested nor featured!

### DISCUSSION THREAD

**Discussion by myndxero** [vladmandic/sdnext#1215] [Ideas]
I'd like a pause button. Pause job. Maybe allow to offload vram into ram or page file, resume whenever. Maybe a similar scenario as dreambooth training? Give it to me baby, uh uh. Pause button, please, I need you in my life.

Now, I USED to STALL out Auto back in the day by just clicking into the middle of the text in the terminal. It certainly allowed GPU to calm down iirc, I don't know about VRAM. I would do it so I could hop on and catch some HOTS matches, then bring up the terminal and hit enter to resume. Vlad's doesn't do that. No idea of Auto's still does that, but it would be a great feature!

I don't know how that might translate, having been able to just click in the middle of the terminal to stop progress in its tracks, but perhaps that offers a thought of where to begin?

SIDE NOTE THOUGHT @ vlad, I saw your changelog about auto 1.2 code being messy to port, now this is hearsay and I've never really investigated, but more than a few times I've seen suggested Auto has a tendency unethically copy paste code and with minimal effort patch it in to work, and that was the foundation of his webUI. No idea about now, 1.9k issue tickets open is like, hmm?

EDIT: Oh, why? Right! Why would this be a good idea? I dunno, for crazy xyz-grid addicts (#gethelp) like myself that generate a billion images creating datasets, finding the right art style, level of detail, whatever, upscale batch, millions of reasons, currently have an 8 hour generation job going and it's 4 hours in and all I want to do is ANYTHING ELSE for a short while and then come back.

**Comment by vladmandic** [comment]
you sort-of have this. there is no way to unload torch itself and it always keeps nearly ~1gb in vram.
but you can go to settings and click unload checkpoint button.
no need to reload it later, it will be automatically reloaded first time you try any operation.

**Reply by myndxero** [reply]
Specifically to pause a giant xyz grid operation? I've 24gb vram nbd there, 3090 TI, but would lik eto free up GPU/CPU when I get bored of waiting on a job. Or restless, or something pops into my head.

**Reply by vladmandic** [reply]
no, this applies when server is already idle and you want to free up resources. i get it, you want a global pause. doable, although not sure how clean. can you open an actual feature request?

**Reply by myndxero** [reply]
Man, all the hard work I put into this post, I thought it was a feature request VS idea...

Anyway, you got it boss!

**Reply by vladmandic** [reply]
It is an idea until it becomes a feature request 😁
